src/clp_ffi_js/ir/StructuredIrStreamReader.cpp

```diff
@@ -132,7 +133,8 @@ auto StructuredIrStreamReader::decode_range(size_t begin_idx, size_t end_idx, bo
             );
             json_str = std::string(cEmptyJsonStr);
         } else {
-            json_str = json_result.value().dump();
+            json_str = json_result.value()
+                               .dump(-1, ' ', false, nlohmann::detail::error_handler_t::replace);
```

Comment:
Since this is in their example: https://json.nlohmann.me/api/basic_json/error_handler_t/

shall we use `nlohmann::json::error_handler_t::replace` alias instead of the one from the `detail` namespace?